lib/reporting/identity_verification_report.rb

```diff
@@ -95,22 +98,41 @@ def as_csv
       csv << []
       csv << ['Metric', '# of Users']
       csv << []
-      csv << ['Started IdV Verification', idv_started]
-      csv << ['Submitted welcome page', idv_doc_auth_welcome_submitted]
-      csv << ['Images uploaded', idv_doc_auth_image_vendor_submitted]
+
+      collect_data(csv, data['all'])
+
+      if @ab_tests.present?
```

Comment:
will `ab_tests?` always be present?

Reply:
to clarify, the reason that I ask, is that we run this report 3x inside of the `ProofingRateReport`, and so any extra processing that is not needed for that report will end up getting run 3x, so I would love if we preserved some version of this report has the extra processing left out